kubernetes/samples/scripts/domain-lifecycle/startDomain.sh

```diff
@@ -67,13 +67,22 @@ function initialize {
 
   validateKubernetesCliAvailable
   validateJqAvailable
+  validateYqAvailable
+
   failIfValidationErrors
 }
 
 initialize
 
 # Get the domain in json format
-domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o json)
+domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o json --ignore-not-found)
+
+if [ -z "${domainJson}" ]; then
+  printError "Domain resource for domain '${domainUid}' not found in namespace '${domainNamespace}'. \
+    This script requires that the introspector job for the specified domain ran \
+    successfully and generated the domain resource. Exiting."
```

Comment:
the second sentence look like it should be removed ("This script..."), as the introspect job does not generate a domain resource...

Reply:
(same problem  in 'stopDomain.sh' too)